### DISCUSSION TITLE
Disable add component button

### DISCUSSION THREAD

**Discussion by wpcreative-jw** [flyntwp/flynt#428] [Q&A] [answered]
Is there anyway to disable the add component button when handing over to a client? 

I think this theme is great to help build sites more rapidly but I wouldnt want clients to have that level of control to add different block components here there and everywhere on the site.

**Comment by steffenbew** [comment, marked as answer]
I don't think disabling the **add component** button is possible with ACF. Hacking it for a certain user role via a hook or JavaScript might be possible.
If we want the components on a page to be fixed, we rather create a separate field group for a specific template and add the components statically, not as a `flexible_content` field.
However, this removes the ease of creating new pages with the flexible content.
